app/views/site/faq.scala

```diff
@@ -357,63 +357,48 @@ object faq:
         ),
         question(
           "make-a-bot",
-          "Make a Lichess bot?",
+          makeABotQ.txt(),
           p(
-            "To learn how to create a ",
-            a(href := "https://lichess.org/blog/WvDNticAAMu_mHKP/welcome-lichess-bots")("Lichess bot"),
-            ", please read ",
-            a(href := "https://lichess.org/@/thibault/blog/how-to-create-a-lichess-bot/FuKyvDuB")(
-              "this blog post"
-            ),
-            "."
+            makeABotA(
+            a(href := "https://lichess.org/blog/WvDNticAAMu_mHKP/welcome-lichess-bots")("Lichess Bot"),
+            a(href := "https://lichess.org/@/thibault/blog/how-to-create-a-lichess-bot/FuKyvDuB")(thisBlogPost())
```

Comment:
This shouldn't be translated as the destination link only has english text. And it's meant for developers.

Reply:
OK why not